### PR TITLE
[DF-5] Home Page

### PR DESCRIPTION
**Pull request by octobrekids**

### CODE REVIEW

**Review by octobrekids — COMMENTED** [review]
Seen

**Review by octobrekids — COMMENTED** [review]
Seen

File: components/Layouts/Home/HomeNavbar/HomeNavbar.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  BtnSideMenu,
+  Logo,
+  Nav,
+} from 'Components/Layouts/Home/HomeNavbar/HomeNavbar.styled';
+import RightNav from 'Components/Layouts/Home/HomeNavbar/RightNav';
+import SideMenu from 'Components/Layouts/Home/HomeNavbar/SideMenu';
+import React, { useState } from 'react';
+
+const HomeNavbar: React.FC = () => {
+  const [visible, setVisible] = useState(false);
```
Comment (yee2542) [inline components/Layouts/Home/HomeNavbar/HomeNavbar.tsx:13]: should able to init `visible` value from props to
```ts
const [visible, setVisible] = useState(props.visible || false)
```

**Review by yee2542 — CHANGES_REQUESTED** [review]

**Comment by yee2542** [comment]
should we call a general button as `CustomButton` ? maybe it can name as `Button`

File: components/Layouts/Home/HomeNavbar/HomeNavbar.tsx (hunk 1 above)
Comment (octobrekids) [reply components/Layouts/Home/HomeNavbar/HomeNavbar.tsx]: Fix here 64a56c474f0bbdadcc9d2498274f69ab1268e332

**Comment by octobrekids** [comment]
> should we call a general button as `CustomButton` ? maybe it can name as `Button`

Fix here 4c5d5344fb69d05dcd1e46487aabf5ad801ca55d

File: pages/index.tsx [hunk 2]
```diff
@@ -1,39 +1,81 @@
...
+const H1 = styled.div`
+  font-size: 5em;
+  color: #ffffff;
+  margin-top: 0;
+  margin-bottom: 0em;
+`;
+
+export const HeaderText = styled.div`
+  color: #ffffff;
+`;
```
Comment (octobrekids) [inline pages/index.tsx:43]: should move to a new file

File: components/Layouts/Home/HomeNavbar/RightNavbar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { UlStyle } from 'Components/Layouts/Home/HomeNavbar/styled';
```
Comment (octobrekids) [inline components/Layouts/Home/HomeNavbar/RightNavbar/index.tsx:1]: Change name `UlStyle` to `Ul`

File: pages/index.tsx [hunk 4]
```diff
@@ -1,39 +1,81 @@
...
-import HomeLayout from 'Components/Layouts/Home/HomeLayout';
+import { Card, Col, Divider, Row } from 'antd';
+import Button from 'Components/Button';
+import HomeLayout from 'Components/Layouts/Home';
+import { APP_FEATURE_CONSTANT } from 'Models/featureCard/constant';
 import React from 'react';
-import style from 'Styles/Home.module.css';
+import styled, { CSSProperties } from 'styled-components';
+import IconSelector from 'Utils/icon-selector';
+
```
Comment (yee2542) [inline pages/index.tsx:8]: Should have `wrapper` to wrap this components

**Review by yee2542 — APPROVED** [review]
Go

File: pages/index.tsx (hunk 2 above)
Comment (octobrekids) [reply pages/index.tsx:43]: Move to Component -> PageWrapper

File: pages/index.tsx [hunk 5]
```diff
@@ -1,39 +1,81 @@
-import { Card, Col, Row } from 'antd';
-import Button from 'Components/Button/Button';
-import HomeLayout from 'Components/Layouts/Home/HomeLayout';
+import { Card, Col, Divider, Row } from 'antd';
```
Comment (octobrekids) [inline pages/index.tsx:1]: Remove unused code

**Review by octobrekids — COMMENTED** [review]
Seen

**Pull request merged**